src/stream/src/executor/global_simple_agg.rs

```diff
@@ -99,6 +106,21 @@ impl<S: StateStore> SimpleAggExecutor<S> {
         let input_info = input.info();
         let schema = generate_agg_schema(input.as_ref(), &agg_calls, None);
 
+        // Create state tables for each agg call.
+        let mut state_tables = Vec::with_capacity(agg_calls.len());
+        for (agg_call, ks) in agg_calls.iter().zip_eq(&keyspace) {
+            let state_table = StateTable::new(
+                ks.clone(),
+                vec![ColumnDesc::unnamed(
+                    ColumnId::new(0),
+                    agg_call.return_type.clone(),
+                )],
+                // Primary key do not includes group key.
+                vec![OrderType::Descending; get_key_len(agg_call)],
```

Comment:
Have to admit that this is not elegant to infer a lot of things in CN: column desc, order type of pk, pk lenth etc... But we have no other choice in current stage.